### PR TITLE
Add TachiyomiSY fork

### PR DESCRIPTION
**Pull request by Soitora**
Adds TachiyomiSY to the website

### CODE REVIEW

**Comment by specterflare** [comment]
Since this fork is like four levels deep in the branching of Tachiyomi (Base -> EH -> AZ -> SY) and that EH is deprecated, have you considered removing EH from the forks page?

File: src/help/guides/creating-backups.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ You can do this by going to <Navigation item="more"/> → <Navigation item="sett
 
 This fork can only restore **MangaDex** entries in a backup. If there are entries from other sources that are a part of that backup then they will not transfer.
 
-#### [TachiyomiAZ](/forks/TachiyomiAZ), and [TachiyomiEH](/forks/TachiyomiEH)
+#### [TachiyomiAZ](/forks/TachiyomiAZ), [TachiyomiSY](/forks/TachiyomiSY), and [TachiyomiEH](/forks/TachiyomiEH)
 
 These two forks have some built-in extensions, namely: **E-Hentai**, **Hitomi.la**, and **nHentai**.
```
Comment (GHOSTBEAR) [inline src/help/guides/creating-backups.md:52]: Isn't it three forks now?

Comment (Soitora) [reply src/help/guides/creating-backups.md:52]: Yes, good catch

Comment (Soitora) [inline src/help/guides/creating-backups.md:52]:
```suggestion
These three forks have some built-in extensions, namely: **E-Hentai**, **Hitomi.la**, and **nHentai**.
```

**Comment by Soitora** [comment]
> Since this fork is like four levels deep in the branching of Tachiyomi (Base -> EH -> AZ -> SY) and that EH is deprecated, have you considered removing EH from the forks page?

I have considered that, for now we'll keep it unless more people agree it should be kept.
I wonder which person will fork SY's version...

File: src/help/guides/creating-backups.md [hunk 2]
```diff
@@ -47,8 +47,8 @@ You can do this by going to <Navigation item="more"/> → <Navigation item="sett
 
 This fork can only restore **MangaDex** entries in a backup. If there are entries from other sources that are a part of that backup then they will not transfer.
 
-#### [TachiyomiAZ](/forks/TachiyomiAZ), and [TachiyomiEH](/forks/TachiyomiEH)
+#### [TachiyomiAZ](/forks/TachiyomiAZ), [TachiyomiSY](/forks/TachiyomiSY), and [TachiyomiEH](/forks/TachiyomiEH)
 
-These two forks have some built-in extensions, namely: **E-Hentai**, **Hitomi.la**, and **nHentai**.
+These three forks have some built-in extensions, namely: **E-Hentai**, **Hitomi.la**, and **nHentai**.
 
 Titles in these extensions on these two forks will not be restored when restoring a backup made by these forks to standard Tachiyomi. It is possible to restore titles to these two forks, but it is not possible to back them up out of these forks.
```
Comment (GHOSTBEAR) [inline src/help/guides/creating-backups.md:54]: Is this also a thing in SY else it should also be `three` here too
```suggestion
Titles in these extensions on these three forks will not be restored when restoring a backup made by these forks to standard Tachiyomi. It is possible to restore titles to these three forks, but it is not possible to back them up out of these forks.
```
Also, I double-check so there is no more of these hidden

Comment (Soitora) [reply src/help/guides/creating-backups.md:54]: Maybe we should just not use any word describing the amount; e.g `these forks` instead of `these X forks`?

Comment (GHOSTBEAR) [reply src/help/guides/creating-backups.md:54]: That could be a solution.
Maybe something along these lines:

> These forks have some built-in extensions, namely: E-Hentai, Hitomi.la, and nHentai.
> 
> Titles from said extensions will not be restored when restoring a backup made by these forks to standard Tachiyomi. It is possible to restore titles to these forks, but it is not possible to back them up out of these forks.

Comment (Soitora) [reply src/help/guides/creating-backups.md:54]: Can you add that as a suggestion so it's one-click commit? Or if you commit directly to my branch. Not at my PC atm

Comment (GHOSTBEAR) [inline src/help/guides/creating-backups.md:54]:
```suggestion
These forks have some built-in extensions, namely: **E-Hentai**, **Hitomi.la**, and **nHentai**.

Titles from said extensions will not be restored when restoring a backup made by these forks to standard Tachiyomi. It is possible to restore titles to these forks, but it is not possible to back them up out of these forks.
```
@Soitora

**Pull request merged**